clone-git-repo@anaximeno/clone-git-repo@anaximeno.nemo_action.in

```diff
@@ -1,7 +1,7 @@
 [Nemo Action]
 _Name=Clone a git repository
 _Comment=Clone a git repository into the current folder.
-Icon-Name=folder-download-symbolic
+Icon-Name=mintinstall-share-symbolic
```

Comment:
Will this icon be available on non-Mint systems?

Reply:
Good point, let me check that.

Reply:
Changed to `emblem-shared-symbolic`, I found it available on more icon themes the `mintinstall-share-symbolic` (which comes with mintinstall).

Thanks!